src/SFA.DAS.FindApprenticeshipTraining.UnitTests/InnerApi/Requests/WhenBuildingTheLocationQueryRequest.cs

```diff
@@ -15,7 +16,19 @@ public void Then_The_Request_Is_Correctly_Built(string baseUrl, string query)
                 BaseUrl = baseUrl
             };
 
-            actual.GetUrl.Should().Be($"{baseUrl}api/locations/search?query={query}");   
+            actual.GetUrl.Should().Be($"{baseUrl}api/search?query={query}");   
+        }
+
+        [Test, AutoData]
+        public void Then_The_Request_Is_Correctly_Encoded_For_Special_Characters(string baseUrl, string query)
```

Comment:
This test also doesn't guarantee a special character needing encoding, see above